registry/github_registry.go

```diff
@@ -38,6 +39,7 @@ type githubRegistry struct {
 	format         common.RegistryFormat
 	credentialName string
 	service        GithubRepositoryService
+	httpClient     *http.Client
```

Comment:
If I'm not mistaken (which I could be since I haven't checked, given that I'm on an airplane and everything is slow), http.Client may be derivable from service.